glide.yaml

```diff
@@ -1,9 +1,15 @@
 package: github.com/openshift/cluster-config-operator
 import:
-  - package: k8s.io/klog
-    version: v1.0.0
+- package: k8s.io/klog
+  version: v1.0.0
```

Comment:
@sttts can you bump this to version v0.4.0 ? (it is what origin use and I think 1.16.x)

Reply:
done